cmd/dartboard/subcommands/deploy.go

```diff
@@ -188,31 +199,32 @@ func chartInstallRancher(r *dart.Dart, rancherImageTag string, cluster *tofu.Clu
 	rancherClusterName := clusterAdd.Public.Name
 	rancherClusterURL := clusterAdd.Public.HTTPSURL
 
-	chartVals := getRancherValsJSON(r.ChartVariables.RancherImageOverride, rancherImageTag, r.ChartVariables.AdminPassword, rancherClusterName, rancherClusterURL, r.ChartVariables.RancherReplicas)
-
-	var extraArgs []string
-	if r.ChartVariables.RancherValues != "" {
-		p, err := writeValuesFile(r.ChartVariables.RancherValues)
-		if err != nil {
-			return fmt.Errorf("writing extra values file: %w", err)
-		}
-		defer os.Remove(p)
-
-		extraArgs = append(extraArgs, "-f", p)
+	var extraEnv []map[string]any
+	extraEnv = []map[string]any{
+		{
+			"name":  "CATTLE_SERVER_URL",
+			"value": rancherClusterURL,
+		},
+		{
+			"name":  "CATTLE_PROMETHEUS_METRICS",
+			"value": "true",
+		},
+		{
+			"name":  "CATTLE_DEV_MODE",
+			"value": "true",
+		},
 	}
+	extraEnv = append(extraEnv, r.ChartVariables.ExtraEnvironmentVariables...)
 
-	return chartInstall(cluster.Kubeconfig, chartRancher, chartVals, extraArgs...)
-}
+	chartVals := getRancherValsJSON(r.ChartVariables.RancherImageOverride, rancherImageTag, r.ChartVariables.AdminPassword, rancherClusterName, extraEnv, r.ChartVariables.RancherReplicas)
 
-func writeValuesFile(content string) (string, error) {
-	p, err := os.CreateTemp("", "values-*.yaml")
-	if err != nil {
-		return "", err
-	}
-	if _, err := io.WriteString(p, content); err != nil {
-		return "", err
+	fmt.Printf("\n\nRANCHER CHART VALS:\n")
+
+	for key, value := range chartVals {
+		fmt.Printf("\t%s = %v\n", key, value)
```

Comment:
This looks like debug stuff. Maybe add to a logger.debug print but probably shouldn't stay in fmt.print

Reply:
In the past this was added as it was a useful way for whoever is running dartboard to vet that the chart values they expected to use are actually being used. But it can be removed if it seems like noise at this point

Reply:
I'll let you tell me -- if this is useful in all cases and isn't just providing noise, then it can stay! I'll approve since this is the last thing, is a nit, and is up to your discretion 👍